### PR TITLE
Shopify Source Documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added documentation for the Shopify source.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: stream-sources/shopify.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+<div class="infoBlock">
+
+<ul>
+<li> RudderStack supports and maps Shopify's <code class="inline-code">customer_create</code> and <code class="inline-code">customer_update</code> events as <code class="inline-code">identify</code> events.</li>
+<li> RudderStack also supports <code class="inline-code">checkout_created</code> and <code class="inline-code">order_update</code> events as the <a href="https://rudderstack.com/docs/rudderstack-api/api-specification/rudderstack-ecommerce-events-specification/">E-commerce events</a>. </li>
+</ul>
+</div>
+
+The following Shopify events are mapped to RudderStack's generic `track` events:
```
Comment (arajguha) [inline stream-sources/shopify.mdx:78]: Can we mention it explicilty that the events in this list are allowed as generic track events. Any other event flowing through would be dropped.

Comment (ameypv-rudder) [reply stream-sources/shopify.mdx:78]: Sure - will add this note.

**Comment by yashasvibajpai** [comment]
We have some more updates in regards to the doc. We don't need to share the Shopify partners process un the documentation. Shared the relevant one already.

**Comment by ameypv-rudder** [comment]
@yashasvibajpai @arajguha @saikumarrs the device mode updates are incorporated in this PR.

**Review by arajguha — APPROVED** [review]

**Comment by arajguha** [comment]
lgtm

File: stream-sources/shopify.mdx [hunk 2]
```diff
@@ -0,0 +1,140 @@
+---
+# slug: "/docs/stream-sources/appcenter"
```
Comment (ishachopra0401) [inline stream-sources/shopify.mdx:2]:
```suggestion
# slug: "/docs/stream-sources/shopify"
```

File: stream-sources/shopify.mdx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  <li><a href="#configuring-the-shopify-source-in-rudderstack">Configuring the Shopify source in RudderStack</a></li>
+  <li><a href="#configuring-the-rudderstack-app-in-your-shopify-store">Configuring the RudderStack app in your Shopify store</a></li>
+</ul>
+</div>
+
+## Configuring the Shopify source in RudderStack
+
+Follow these steps to set up your Shopify source in the RudderStack dashboard:
+
+- Go to your [**RudderStack dashboard**](https://app.rudderstack.com/) and click on **Add Source**. From the list of **Event Stream** sources, select **Shopify**, as shown:
```
Comment (ishachopra0401) [inline stream-sources/shopify.mdx:29]: numbered list may be helpful

File: stream-sources/shopify.mdx [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+<img src="../assets/event-stream-sources/shopify-2.png" alt="Shopify source webhook URL" />
+
+- Finally, connect the source to your desired destinations.
+
+## Configuring the RudderStack app in your Shopify store
+
+To complete the configuration, you will need to add the webhook URL copied above in the RudderStack app installed in your Shopify store. Follow these steps to do so:
+
+- Go to the [**admin dashboard**](https://accounts.shopify.com/store-login) of your Shopify store.
```
Comment (ishachopra0401) [inline stream-sources/shopify.mdx:45]: numbered list may be helpful

File: stream-sources/shopify.mdx [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+- Your Shopify source is now configured. Note the source **Write key**. This will be required later while configuring the RudderStack app on your Shopify store.
+
+<img src="../assets/event-stream-sources/shopify-2.png" alt="Shopify source webhook URL" />
+
+- Finally, connect the source to your desired destinations.
+
+## Configuring the RudderStack app in your Shopify store
+
+To complete the configuration, you will need to add the webhook URL copied above in the RudderStack app installed in your Shopify store. Follow these steps to do so:
```
Comment (ishachopra0401) [inline stream-sources/shopify.mdx:43]: Not quite sure where we copied the webhook URL

File: stream-sources/shopify.mdx [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+- Go to the [**admin dashboard**](https://accounts.shopify.com/store-login) of your Shopify store.
+
+- In the left sidebar, click on **Apps**. Then, click on **Customize your store**, as shown:
+
+<img src="../assets/event-stream-sources/shopify-3.png" alt="Customizing Shopify store" />
+
+- Then, search for RudderStack, as shown:
```
Comment (ishachopra0401) [inline stream-sources/shopify.mdx:51]:
```suggestion
- Search for RudderStack, as shown:
```
then is repeating in above and below steps.

File: stream-sources/shopify.mdx [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+<img src="../assets/event-stream-sources/shopify-4.png" alt="RudderStack app search" />
+
+- In the search results, click on the RudderStack app. Then click on **Add app**. 
+
+- After installation, you should see the app in the **Installed apps** section, as shown:
```
Comment (ishachopra0401) [inline stream-sources/shopify.mdx:57]:
```suggestion
- After installation, you should be able to see the app in the **Installed apps** section, as shown:
```

**Review by ishachopra0401 — CHANGES_REQUESTED** [review]

File: stream-sources/shopify.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply stream-sources/shopify.mdx:2]: this won't be rendered anyway :)

**Pull request closed without merging**